src/h5web/vis-packs/core/heatmap/ColorBar.module.css

```diff
@@ -51,6 +51,7 @@
   justify-content: center;
   line-height: 1;
   font-size: 0.875em;
+  white-space: nowrap;
```

Comment:
For consistency, I've changed the formatting of the bounds on the colour map to use exponent notation, and the browser wraps the value at `+/-` if there's not enough space. We want the bounds to overflow without wrapping.